doc/testing.md

```diff
@@ -66,3 +64,31 @@ Also test watcher can be launched. It will re-run the entire test suite when any
 ```
 
 Check [component tests doc](./component-tests-overview.md) for more.
+
+## Unit tests only
+
+To run unit tests:
+
+```
+   make unit-test
+```
+
+Also test watcher can be launched. It will re-run the entire test suite when any file is modified
+
+```
+   make unit-test WATCH=true
+```
+
+## Integration tests only
+
+To run integration tests:
+
+```
+   make integration-test
+```
+
+Also test watcher can be launched. It will re-run the entire test suite when any file is modified
+
+```
+   make integration-test WATCH=true
```

Comment:
It's unnecessary to indent code blocks if they're already inside triple backticks.